### PR TITLE
Db utils

### PR DESCRIPTION
**Pull request by wslulciuc**

### CODE REVIEW

File: src/test/java/marquez/db/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testToUuidOrNull_throwsException_onBlankUuidString() {
+    final String blankUuidString = " ";
+    Utils.toUuidOrNull(blankUuidString);
+  }
+
+  @Test
+  public void testToListOrEmpty() throws SQLException {
```
Comment (ashulmanWeWork) [inline src/test/java/marquez/db/UtilsTest.java]: I don't think this function name matches the test (at least the empty list part). It's important to add a test for an empty list, and I don't think I see that case exercised. Can we add it?
